PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -415,7 +415,9 @@ extension PocketSource {
     }
 
     public func fetchDetails(for recommendation: Recommendation) async throws {
+        Log.breadcrumb(category: "detail-loading", level: .debug, message: "Loading details for Recomendation: \(String(describing: recommendation.remoteID))")
```

Comment:
Adding a breadcrumb here for when we load a detail view, I think we should start adding these to drilldown calls for future debugging.